package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "loopback-component-jsonapi",
+  "version":"0.0.1",
```

Comment:
We use semantic release which dynamically creates the correct version number on publish so we purposely don't keep this in package.json

https://github.com/semantic-release/semantic-release